dask/delayed.py

```diff
@@ -267,11 +288,14 @@ def delayed(obj, name=None, pure=False, nout=None, traverse=True):
     >>> a.count(2, pure=True).key == a.count(2, pure=True).key
     True
 
-    As with function calls, method calls also support the ``dask_key_name``
-    keyword:
+    As with function calls, method calls also respect the global
+    ``delayed_pure`` setting and support the ``dask_key_name`` keyword:
 
     >>> a.count(2, dask_key_name="count_2")
-    Delayed("count_2")
+    Delayed('count_2')
+    >>> with dask.set_options(delayed_pure=True):
+    ...     print(a.count(2).key == a.count(2).key)
+    True
     """
```

Comment:
This is true, but it's already documented. Is this not clear enough? Suggestions for improvement? https://github.com/dask/dask/blob/master/dask/delayed.py#L243-L246.